### PR TITLE
ResourceGroup - Add Support for getGroupConfiguration

### PR DESCRIPTION
**Pull request by usmangani1**
ResourceGroup - Add Support for getGroupConfiguration

### CODE REVIEW

File: moto/resourcegroups/responses.py [hunk 1]
```diff
@@ -22,8 +22,13 @@ def create_group(self):
...
         resource_query = self._get_param("ResourceQuery")
         tags = self._get_param("Tags")
+        configuration = self._get_param("Configuration")
         group = self.resourcegroups_backend.create_group(
-            name=name, description=description, resource_query=resource_query, tags=tags
+            name=name,
+            description=description,
+            resource_query=resource_query,
+            tags=tags,
+            configuration=configuration,
```
Comment (bblommers) [inline moto/resourcegroups/responses.py:31]: This should have it's own test, to confirm that we can create a group that already contains a Configuration

File: moto/resourcegroups/models.py [hunk 2]
```diff
@@ -350,6 +359,16 @@ def update_group_query(self, group_name, resource_query):
         self.groups.by_name[group_name].resource_query = resource_query
         return self.groups.by_name[group_name]
 
+    def get_group_configuration(self, group_name):
+        group = self.groups.by_name.get(group_name)
+        configuration = group.configuration
+        return configuration
+
+    def put_group_configuration(self, group_name, configuration):
+        if configuration:
```
Comment (bblommers) [inline moto/resourcegroups/models.py:368]: Is this a valid scenario? I'm assuming that the Configuration-param is required when calling this method

**Review by bblommers — CHANGES_REQUESTED** [review]
Hi @usmangani1, please see my comments.

One of the failing tests (test_s3_server_post_without_content_length) was due to an issue with our CI - please merge in the latest changes from master if this test is still failing on the next commit

File: moto/resourcegroups/responses.py (hunk 1 above)
Comment (usmangani1) [reply moto/resourcegroups/responses.py:31]: done

File: moto/resourcegroups/models.py (hunk 2 above)
Comment (usmangani1) [reply moto/resourcegroups/models.py]: Done

**Review by bblommers — APPROVED** [review]
LGTM - thank you!

**Pull request merged**